docs/imagecustomizer/api/configuration/injectArtifactMetadata.md

```diff
@@ -58,16 +58,30 @@ Added in v0.14.
 
 Required.
 
-Path to the signed artifact file to be injected. This path may be relative to the
+Path to the artifact file to be injected. This path may be relative to the
 `inject-files.yaml` config file or an absolute path.
 
+The file at this path should be initially the unsigned artifact extracted during
+the output artifacts phase. Users should replace this file with the signed version
+before running the injection command.
+
+Artifacts are organized in subdirectories matching their type (e.g., `ukis/`, `shim/`,
+`systemd-boot/`, `verity-hash/`).
+
 Added in v0.14.
 
-## `unsignedSource` [string]
+## `type` [string]
+
+Required.
 
-Optional.
+The type of artifact being injected. This field helps users identify which signing
+process to apply to each artifact.
 
-Path to the original unsigned artifact (if available). This field is for informational
-or auditing purposes only — it is not used during injection.
+Supported values:
```

Comment:
It might be good to provide a link to the `.output.artifacts.items` API, since the supported values are equivalent.